examples/calculate_eventdisplay_irfs.py

```diff
@@ -109,6 +109,8 @@ def main():
     background = table.vstack(
         [particles["proton"]["events"], particles["electron"]["events"]]
     )
+
+    INITIAL_GH_CUT = np.quantile(gammas['gh_score'], (1 - INITIAL_GH_CUT_EFFICENCY))
```

Comment:
wasn't this supposed to be a simple fixed cut?

Reply:
Yes, it is a single cut. The global cut which has 40 % efficiency. This calculates a single cut with that efficiency.